.buildkite/pipeline.full.yml

```diff
@@ -595,18 +601,21 @@ steps:
           - "features/fixtures/maze_runner/mazerunner_2021.3.13f1.ipa"
         upload:
           - "maze_output/**/*"
-      docker-compose#v3.7.0:
-        pull: maze-runner
-        run: maze-runner
+      docker-compose#v4.7.0:
+        pull: maze-runner-bitbar
+        run: maze-runner-bitbar
+        service-ports: true
         command:
-          - "--app=/app/features/fixtures/maze_runner/mazerunner_2021.3.13f1.ipa"
-          - "--farm=bs"
-          - "--device=IOS_15"
-          - "--fail-fast"
           - "features/csharp"
           - "features/ios"
-    concurrency: 5
-    concurrency_group: browserstack-app
+          - "--app=features/fixtures/maze_runner/mazerunner_2021.3.13f1.ipa"
+          - "--farm=bb"
+          - "--device=IOS_11|IOS_12|IOS_13|IOS_14|IOS_15"
```

Comment:
Maybe best to remove the iOS 11 a there are no such devices on BitBar.  Could cause surprising failures in the future should one be added.